pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "pydle"
+version = "0.10.0"
```

Comment:
It was discussed earlier that this needed to be a full point release, so we need to undo an earlier change that declared this just a minor release.

Reply:
You're right. Pushed a commit to bump the version meta to 1.0.0